azure-kusto-data/tests/test_e2e_data.py

```diff
@@ -23,6 +23,13 @@
 from azure.kusto.data.streaming_response import FrameType
 
 
+@pytest.fixture(scope="module")
+def event_loop_policy(request):
+    if platform.system() == "Windows":
+        return asyncio.WindowsSelectorEventLoopPolicy()
```

Comment:
I'm curious why use `WindowsSelectorEventLoopPolicy` for Windows, instead of always using `DefaultEventLoopPolicy`?

(And if we always used `DefaultEventLoopPolicy`, there'd be no need for this method altogether.)

Reply:
It's a problem upstream with pytest-asyncio related to closing the loop when testing ends. Then I think they fixed it and then there was a regression, so I added a comment and leaving it for now.